modules/mobile/app/models/mobile/v0/adapters/va_appointments.rb

```diff
@@ -94,7 +94,7 @@ def build_appointment_model(appointment_hash)
                         Mobile::V0::Appointment.encode_cancel_id(
                           start_date_local: start_date_local,
                           clinic_id: appointment_hash[:clinic_id],
-                          facility_id: facility_id,
+                          facility_id: Mobile::V0::Appointment.convert_to_non_prod_id!(facility_id),
```

Comment:
Seems like this is one the parts we're fairly confident about since it's currently broken.

Reply:
yup, this is the only change in behavior for this PR. other scheduling endpoints still need to be addressed but wanted to unblock QA.